src/bindings/jna/README.md

```diff
@@ -1,43 +1,121 @@
-A full java binding using JNA.
+A full java binding using JNA. This binding requires elektra to be installed on 
+the system to work.
 
 ## Usage ##
 
+To use the bindings in a java project, we have to include the jar file 
+libelektra-0.8.19.jar in the project. This jar is created upon build, if you 
```

Comment:
ok i've done the kiss solution and just call it *version*, i guess people will get it. To be absolutely precise i've even added a small sentence that its equivalent to the elektra version.